content/2016.09.jedit-trackers-with-kinto-en.md

```diff
@@ -0,0 +1,403 @@
+Title: Getting jEdit Trackers off the Ground using Kinto and Kinto.js
```

Comment:
I wanted to allude to kinto the yellow cloud and also starting to use kinto.
But your proposal "Getting an overview of bugs with Kinto and Kinto.js" is fine.